### PR TITLE
Add more details to Tech FAQs

### PR DESCRIPTION
**Pull request by jhaaaa**
- Adds more details to the Technical FAQs
- Nit: Fixes language that attributes XRP Ledger or `rippled` functionality to Ripple, the company. The functionality is now attributed to the appropriate technology, XRP Ledger or `rippled`, for example.

### CODE REVIEW

**Comment by jhaaaa** [comment]
@mDuo13 - When you have a moment, I'd appreciate a sanity check on this. Attributing the base58 format to XRP Ledger feels wordy in place. On the bright side, I feel that the docs now call out a key detail about the base58 format. As always, open to your feedback.

@nbougalis - If you have the time and interest, please feel free to provide your feedback, esp on the details added to the Tech FAQs.

File: content/concepts/introduction/technical-faq.md [hunk 1]
```diff
@@ -77,8 +77,17 @@ Ripple is committed to monitoring and reporting any AML flags across the XRP Led
 
 #### What is Ripple’s process for reviewing third-party code contributions before they go live in the master codebase?
 
-Ripple controls who has access to modify official versions of the `rippled` server source code, and it thoroughly audits all code.
+The code contribution process starts with a developer opening a pull request against Ripple's `rippled` repo. This pull request triggers automated unit and integration tests, as well as code reviews by several developers who, typically, have significant expertise in the area of code that the pull request is changing.
 
+Once the pull request passes automated tests and receives approvals from reviewers, a trusted <a href="https://opensource.guide/best-practices/" target="_blank">maintainer of the repo <i class="fa fa-external-link" aria-hidden="true"></i></a><!--#{ fix md highlighting_ #}--> can stage it for inclusion in the next beta.
```
Comment (mDuo13) [inline content/concepts/introduction/technical-faq.md]: This HTML syntax was bugging so much, I made a filter so we never have to use it again. https://github.com/ripple/ripple-dev-portal/pull/510

I recommend switching these to use standard markdown syntax now. When the filter is merged, it'll automatically mark them as intended.

**Review by mDuo13 — APPROVED** [review]
Looks great. I would change the external link syntax to be standard markdown link syntax, but otherwise, no complaints.

**Comment by mDuo13** [comment]
The changes you made miiiight conflict with the ones I made in #510 but we can work out the merge conflicts if we get any

File: content/concepts/introduction/technical-faq.md (hunk 1 above)
Comment (jhaaaa) [reply content/concepts/introduction/technical-faq.md]: Genius! 🙇‍♀️ Thank you for working your magic 🌟 to relieve the eyestrain of the HTML. I've switched them to md format.

**Pull request merged**